examples/00-fluent/ahmed_body_workflow.py

```diff
@@ -213,7 +221,7 @@
 #######################################################################################
 # Switch to the Solver Mode
 # =====================================================================================
-session = session.switch_to_solver()
+session: Solver = session.switch_to_solver()
```

Comment:
Do we need this change in script? I think this should be applicable in type hint only.

Reply:
@hpohekar Maybe this is good for readability?